### ISSUE TITLE
Data download

### ISSUE THREAD

**Issue by yifliu3** [Jun-CEN/Open_world_3D_semantic_segmentation#8]
Hi, thanks for your great work!

I'd like to reproduce the results implemented in the paper, however, the dataset seems rather large, and can I just download part of the SemanticKitti and Nuscenes dataset?

**Comment by yifliu3** [comment]
I find that in the build function, the code chooses the v1.0-test split of the whole part. So we only need to download this part, right?
```    
if "nusc" in dataset_config['pc_dataset_type']:
        from nuscenes import NuScenes
        nusc = NuScenes(version='v1.0-test', dataroot=data_path, verbose=True)
```

**Comment by Jun-CEN** [comment]
Hi,

Thanks for your interest. If you want to reproduce the results, you should download the whole dataset because we use the whole dataset to train the model. If you want to evaluate the results, maybe the validation set or test set is enough.

**Comment by yifliu3** [comment]
Hi,

Thanks for your prompt reply. So I also need to download the whole dataset for NuScenes, right?

**Comment by Jun-CEN** [comment]
Yes.

**Comment by yifliu3** [comment]
Thanks a lot!
